### ISSUE TITLE
docker-compose 1.5 error with ADD and a script url

### ISSUE THREAD

**Issue by ghost** [Bitergia/fiware-chanchan-docker#4]
I'm using a 1.5.0 docker-compose version with your pep proxy (https://github.com/Bitergia/fiware-chanchan-docker/blob/master/compose/pep-wilma.yml).

idm and pep are giving me the same error on their Dockerfile's line:
`ADD https://raw.githubusercontent.com/Bitergia/docker/master/utils/entrypoint-common.sh /`

```
Step 17 : ADD https://raw.githubusercontent.com/Bitergia/docker/master/utils/entrypoint-common.sh /
Traceback (most recent call last):
  File "/usr/local/bin/docker-compose", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/compose/cli/main.py", line 54, in main
    command.sys_dispatch()
  File "/usr/local/lib/python2.7/dist-packages/compose/cli/docopt_command.py", line 23, in sys_dispatch
    self.dispatch(sys.argv[1:], None)
  File "/usr/local/lib/python2.7/dist-packages/compose/cli/docopt_command.py", line 26, in dispatch
    self.perform_command(*self.parse(argv, global_options))
  File "/usr/local/lib/python2.7/dist-packages/compose/cli/main.py", line 170, in perform_command
    handler(project, command_options)
  File "/usr/local/lib/python2.7/dist-packages/compose/cli/main.py", line 188, in build
    project.build(service_names=options['SERVICE'], no_cache=no_cache, pull=pull)
  File "/usr/local/lib/python2.7/dist-packages/compose/project.py", line 284, in build
    service.build(no_cache, pull)
  File "/usr/local/lib/python2.7/dist-packages/compose/service.py", line 731, in build
    all_events = stream_output(build_output, sys.stdout)
  File "/usr/local/lib/python2.7/dist-packages/compose/progress_stream.py", line 15, in stream_output
    for event in utils.json_stream(output):
  File "/usr/local/lib/python2.7/dist-packages/compose/utils.py", line 131, in split_buffer
    yield decoder(buffered)
  File "/usr/lib/python2.7/json/decoder.py", line 369, in decode
    raise ValueError(errmsg("Extra data", s, end, len(s)))
ValueError: Extra data: line 2 column 1 - line 18 column 1 (char 4 - 785)
```

I changed that URL to a local file and it works, just like:
`COPY entrypoint-common.sh /`

**Comment by dmabtrg** [comment]
Can you show us the full command you are using to get that error?  That ADD line is meant to be used only when building the images, and our compose yml is configured to download the images from the hub, not to build them.

**Comment by ghost** [comment]
I changed the docker compose to built the images myself.

I don't understand what you mean with full command, that command:
`ADD https://raw.githubusercontent.com/Bitergia/docker/master/utils/entrypoint-common.sh /` gave me that output with docker compose v1.5

**Comment by dmabtrg** [comment]
I've been able to reproduce the error with the latest docker-compose 1.5.1 from Debian unstable.  I'm going to do some extra tests and probably open an issue on the docker-compose repository.

**Comment by dmabtrg** [comment]
docker/compose#2398

**Comment by dmabtrg** [comment]
A fix for this issue has been merged and is scheduled for docker compose 1.5.2, so I'm closing this issue.
